README.md

```diff
@@ -113,16 +113,27 @@ Libraries for publishing to NPM should have their directory names prefixed with
 
 Apps should have their directory names prefixed with `app-`, e.g. `/project` becomes `/app-project`.
 
-## Production deployment
+## Deployment
 
 Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image (`zooniverse/front-end-monorepo`) is built with the top-level Dockerfile, which installs and builds production versions of the `lib-` packages. 
 
-That image is used as a base image to build `zooniverse/fe-project` and `zooniverse/fe-content-pages` images, from the Dockerfiles in `packages/app-project` and `packages/app-content-pages`. Those images run production (static) builds of the NextJS apps and are then deployed to Kubernetes.
+Merges to master deploy to a staging Kubernetes instance that uses Panoptes production. This is used for manual end to end behavior testing for new code and design reviews. `https://frontend.preview.zooniverse.org/projects/:project-owner/:project-name/` proxy redirects to the new NextJS app while the rest of sub-domain redirects to PFE. Staging projects can be loaded by adding this query param to the URL: `?env=staging`.
 
-The latest commit to master is deployed to https://frontend.preview.zooniverse.org
-The `production-release` git tag is deployed to https://www.zooniverse.org
+Deployments to a production Kubernetes instance are triggered by committing a `production-release` git tag on master. This can either be done using the git CLI or using the lita deploy command on slack. `https://www.zooniverse.org/projects/:project-owner/:project-name/classify` proxy redirects to the new NextJS app while the rest of the domain redirects to PFE. Currently the only project that is configured to do this is Planet Hunters TESS. Eventually more projects will migrate when they migrate to the new classifier.
+
+More information is available in [ADR 12](docs/arch/adr-12.md) and [ADR 17](docs/arch/adr-17.md)
+
+### Environment variables
+
+- `PANOPTES_ENV`: sets which Panoptes API endpoint to use. Setting to `production` will use `https://www.zooniverse.org/api` and `staging` to `https://panoptes-staging.zooniverse.org/api`. The yarn build scripts for libraries default to production and overrides the environment to be production for the apps. 
+- `NODE_ENV`: sets the environment for library builds.
+
+### Docker images
+
+- `zooniverse/front-end-monorepo`: Used as a base image to build `zooniverse/fe-project` and `zooniverse/fe-content-pages` images from the Dockerfiles in `packages/app-project` and `packages/app-content-pages`
```

Comment:
Built from the Dockerfile in the root directory. It runs `yarn install` and builds the `lib-` packages and static (production) apps.

Reply:
#1109 moves the static builds back into the Dockerfiles for the respective NextJS apps.